apis/Google.Cloud.Asset.V1Beta1/Google.Cloud.Asset.V1Beta1/AssetService.cs

```diff
@@ -646,9 +651,12 @@ public sealed partial class BatchGetAssetsHistoryRequest : pb::IMessage<BatchGet
     public const int ReadTimeWindowFieldNumber = 4;
     private global::Google.Cloud.Asset.V1Beta1.TimeWindow readTimeWindow_;
     /// <summary>
-    /// Required. The time window for the asset history. The start time is
-    /// required. The returned results contain all temporal assets whose time
-    /// window overlap with read_time_window.
+    /// Optional. The time window for the asset history. Both start_time and
```

Comment:
Just flagging: I think this is breaking, because now the `start_time` if set needs to be after a given date, as where before it needed to be set but not in a particular range. This will break everyone who has `start_time` set before that given date.